kubernetes-manifests/recommendationservice.yaml

```diff
@@ -63,11 +63,11 @@ spec:
           value: "1"
         resources:
           requests:
-            cpu: 100m
-            memory: 220Mi
+            cpu: 10m
+            memory: 10Mi
           limits:
-            cpu: 200m
-            memory: 450Mi
+            cpu: 30m
+            memory: 30Mi
```

Comment:
Getting `OOMKilled` with this, let's have `70Mi` in `limits` and `50Mi` in `requests` like the other Python service `emailservice` has?